### PR TITLE
Webpack dev server fix on all particles.

### PR DESCRIPTION
**Pull request by ben-burlingham**
No need to publish; patches are only in sandbox.

### CODE REVIEW

File: tiered-select/README.md [hunk 1]
```diff
@@ -4,18 +4,22 @@ A tiered select.
 
 # Change Log
 
+<<<<<<< HEAD
```
Comment (mcclayton) [inline tiered-select/README.md]: Merge tag still here.

File: bulk-editor/package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "sass-loader": "^6.0.6",
     "style-loader": "^0.18.2",
     "webpack": "^3.4.1",
-    "webpack-dev-server": "^2.6.1",
```
Comment (mcclayton) [inline bulk-editor/package.json]: Did you want to give this particle a `PATCH` version bump as well?

Comment (ben-burlingham) [reply bulk-editor/package.json]: Right yeah I should have specified.  The bulk-editor had a separate issue that causes the sandbox to break so I was going to fix that in a separate branch, so there's not a broken release.  Is that cool?

Comment (mcclayton) [reply bulk-editor/package.json]: 👍

**Pull request merged**